src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
+		<Description>Microsoft Azure Management DataBox library</Description>
+		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
+		<Version>1.0.0-preview</Version>
+		<PackageTags>Microsoft Azure DataBox management;DataBox;DataBox management;</PackageTags>
```

Comment:
@kushalvms all tags with spaces become individual tags on nuget.org
Suggestion:
I would recommend to limit your tags to limited ones that your customer is going to use.
For e.g. if someone wants to search for databox, they are not going to start their search by writing Azure. Rather use AzureDatabox as 1 word.